src/GameCoupling.ts

```diff
@@ -0,0 +1,103 @@
+export class Game {
+    private _lastSymbol: string = ' ';
+    private _board: Board = new Board();
+
+    public Play(symbol: string, x: number, y: number) : void {
```

Comment:
External coupling too: the consumer must know the symbols protocol 

Sovling: change it to data coupling playX(1,0) playO(1,2)